Author: aszlig <aszlig@nix.build>
Date:   Fri Sep 21 21:31:07 2018 +0200

    Fix bcrypt dependency in package.json
    
    The package.json in bcrypt states that node-pre-gyp is bundled, which it
    is for the upstream release package via NPM.
    
    However, a while ago this has been switched to a forked branch of
    another GitHub repository which obviously doesn't come with that bundled
    dependency, given that it's not a release tarball but the source tree.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: fix-bcrypt-dependency.patch

diff --git a/package.json b/package.json
index 7446b6746..f786c506e 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "babel-preset-es2015": "^6.6.0",
     "babel-register": "^6.6.0",
     "babel-runtime": "^6.11.6",
-    "bcrypt": "github:MylesBorins/node.bcrypt.js#update-nan",
+    "bcrypt": "^3.0.1",
     "body-parser": "^1.18.3",
     "bootstrap": "^4.1.3",
     "bootstrap-vue": "^2.0.0-rc.9",
